src/Gitlab/Client.ts

```diff
@@ -58,22 +57,45 @@ export class GitLabClient {
     }
 
     private async editIssue(opts: EditIssueOpts): Promise<CreateIssueResponse> {
-        return (await axios.put(`api/v4/projects/${opts.id}/issues/${opts.issue_iid}`, opts, this.defaultConfig)).data;
+        return (await axios.put(`api/v4/projects/${encodeURIComponent(opts.id)}/issues/${opts.issue_iid}`, opts, this.defaultConfig)).data;
     }
 
 
     private async getProject(id: ProjectId): Promise<GetProjectResponse> {
         try {
-            return (await axios.get(`api/v4/projects/${getProjectId(id)}`, this.defaultConfig)).data;
+            return (await axios.get(`api/v4/projects/${encodeURIComponent(id)}`, this.defaultConfig)).data;
         } catch (ex) {
             log.warn(`Failed to get project:`, ex);
             throw ex;
         }
     }
 
+    private async listProjects(minAccess: AccessLevel, inGroup?: ProjectId, idAfter?: number, search?: string): Promise<SimpleProject[]> {
+        try {
+            const path = inGroup ? `api/v4/groups/${encodeURIComponent(inGroup)}/projects` : 'api/v4/projects';
+            return (await axios.get(path, {
+                ...this.defaultConfig,
+                params: {
+                    archived: false,
+                    min_access_level: minAccess,
+                    simple: true,
+                    pagination: "keyset",
+                    per_page: 50,
+                    order_by: "id",
+                    sort: "asc",
+                    id_after: idAfter,
+                    search,
+                }
+                })).data;
```

Comment:
Is the linter inactive or `intent` not configured properly?
```suggestion
            })).data;
```